perceval/rendering/pdisplay.py

```diff
@@ -297,16 +397,9 @@ def pdisplay(o, output_format: Format = None, **opts):
     if output_format is None:
         output_format = _default_output_format(o)
     res = _pdisplay(o, output_format=output_format, **opts)
+
     if res is None:
-        opts_simple = {}
-        if "precision" in opts:
-            opts_simple["precision"] = opts["precision"]
-        if isinstance(o, (int, float)):
-            res = simple_float(o, **opts_simple)[1]
-        elif isinstance(o, complex):
-            res = simple_complex(o, **opts_simple)[1]
-        else:
-            raise RuntimeError("pdisplay not defined for type %s" % type(o))
+        return
```

Comment:
I would have kept `raise RuntimeError("pdisplay not defined for type %s" % type(o))` instead of the return

Reply:
Moved the exception as NotImplementedError in a dispatched _pdisplay(object) function. So this comment is resolved